pkg/kubelet/lifecycle/handlers.go

```diff
@@ -197,8 +197,8 @@ func (a *noNewPrivsAdmitHandler) Admit(attrs *PodAdmitAttributes) PodAdmitResult
 		return PodAdmitResult{Admit: true}
 	}
 
-	// Always admit runtimes except docker.
-	if a.Runtime.Type() != kubetypes.DockerContainerRuntime {
+	// Always admit runtimes is remote
+	if a.Runtime.Type() == kubetypes.RemoteContainerRuntime {
 		return PodAdmitResult{Admit: true}
 	}
```

Comment:
This will always be `true` so just return and remove the rest of the logic as below.
```suggestion
	return PodAdmitResult{Admit: true}
```